UserInterface.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Weather_Monitoring_System.DataParsers;
+
+namespace Weather_Monitoring_System
+{
+    public class UserInterface
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Welcom To Weather Monitoring System");
+            Console.WriteLine("Enter the weather data usign JSON or XML format\n");
+
+            var weatherDataInput = Console.ReadLine();
+
+            if (weatherDataInput.StartsWith('{'))
+            {
+                var weatherService = new WeatherService(new JsonWeatherDataParser());
```

Comment:
Again, we shouldn't init obj from the services in this way. Please consider applying the IoC principle.